lookout/style/format/features.py

```diff
@@ -151,6 +148,31 @@ def __init__(self, language: str, siblings_window: int = 5, parents_depth: int =
         self.tokens = importlib.import_module("lookout.style.format.langs.%s.tokens" % language)
         self.roles = importlib.import_module("lookout.style.format.langs.%s.roles" % language)
 
+        # Order is important and should be consistent with _inplace_write_vnode_features function
+        # where features generation happens.
+        self.feature_layout = OrderedDict([
```

Comment:
Better to make it protected and add a read-only property

Reply:
It should be better to use an Enum for the keys since they won't change much and it's much less type-error prone

Reply:
Looks good to me! Maybe avoiding self though (current is an option).